SJ3.PAS

```diff
@@ -2453,10 +2453,11 @@ begin
         if (SDLPort.KeyPressed) then
          begin
           SDLPort.WaitForKeyPress(ch,ch2);
+          ch:=upcase(ch);
          end;
         if (ch=#0) and (Ch2=#68) then begin cupslut:=true; ch:=#27; end;
         if (Ch=#27) or (Ch=#13) then Out:=True;
```

Comment:
Does the fix affect these other checks here?

Reply:
[I've made a quick test](https://ideone.com/6DLM95) and `upcase` seems to return unmodified character code if it's outside the `'a'..'z'` range, so it shouldn't break anything here. And this conversion is done in two other places in this very file in the exact same manner, so either way we're at least consistent. I see there are also other places in code where `upcase` is called for the specific check only, but since those two previously mentioned seem to work fine I don't think there will be any problem.